packages/commerce-sdk-react/setup-jest.js

```diff
@@ -42,6 +35,3 @@ Object.defineProperty(window, 'localStorage', {
 global.afterEach(() => {
     nock.cleanAll()
 })
-global.afterAll(() => {
-    nock.restore()
```

Comment:
What was the reason for not calling `nock.restore`? Last time we found it necessary to call it: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/792#discussion_r1005070238

Reply:
because we don't need it anymore with `mockAuthCalls` / `nock.back` removed. `nock.cleanAll` does everything we need already.

Reply:
Agree @kevinxh , we don't want to to use `nock.restore` as it pertains to the recordings which are being dropped